### PR TITLE
[experimental/python] Clean-up

### PR DESCRIPTION
**Pull request by khalatepradnya**
* Drop `qlist`
* Re-enable working test(s)
* Applied code formatting

### Description
https://github.com/NVIDIA/cuda-quantum/issues/1157

### CODE REVIEW

File: python/cudaq/kernels/uccsd.py [hunk 1]
```diff
@@ -109,8 +109,8 @@ def uccsd_num_parameters(n_electrons, n_qubits):
     return sum((singles, doubles, total))
 
 
-def single_excitation_gate(kernel, qubits: cudaq.qlist, p_occ: int, q_virt: int,
-                           theta: list[float]):
+def single_excitation_gate(kernel, qubits: cudaq.qvector, p_occ: int,
```
Comment (amccaskey) [inline python/cudaq/kernels/uccsd.py:112]: qvector -> qview

Comment (khalatepradnya) [reply python/cudaq/kernels/uccsd.py]: Fixed in 5a10fb4

File: python/tests/kernel/test_ir_operations.py [hunk 2]
```diff
@@ -13,22 +13,25 @@
...
 @pytest.fixture(autouse=True)
 def do_something():
     cudaq.enable_jit()
     yield
-    if cudaq.is_jit_enabled(): cudaq.__clearKernelRegistries()
+    if cudaq.is_jit_enabled():
+        cudaq.__clearKernelRegistries()
     cudaq.disable_jit()
 
+
```
Comment (schweitzpgi) [inline python/tests/kernel/test_ir_operations.py:29]: Edification question: I notice there are a lot of blank lines added in this PR. Is this something in a python coding convention someplace?

**Review by schweitzpgi — COMMENTED** [review]
LGTM

File: python/tests/kernel/test_ir_operations.py (hunk 2 above)
Comment (khalatepradnya) [reply python/tests/kernel/test_ir_operations.py:29]: Yep, I applied code formatting with `yapf` to all changed files, including tests (usually tests are skipped, but I prefer to have it uniform.)

**Pull request merged**